modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,430 @@
+
+The Manual Approval Gate controller includes the following features:
+
+* You must set the following parameters in the `ApprovalTask` specification:
+** `approvers`: The users who can approve or reject the `approvalTask` to unblock the pipeline
+** `numberOfApprovalsRequired`: The number of approvals required to unblock the pipeline
+** `description`: Description of the `approvalTask` that {pipelines-shortname} displays to the users
+* Support for approval from multiple users:
+** The approval requires the configured minimum number of approvals front he configured users. Until this number is reached, the approval task does not finish and its `approvalState` value remains `pending`.
+** If any one approver rejects the approval, the task controller changes the `approvalState` of the task to `rejected` and the pipeline run fails.
+** If one user approves the task but the configured number of approvals is still not reached, the same user can change to rejecting the task and the pipeline run fails.
+* Users can provide approval using the `tkn-approvaltask` CLI and the OpenShift web console.
+* Users can add messages while approving or rejecting the `approvalTask`. {pipelines-shortname} records these messages in the YAML manifest of the `pipelineRun` CR.
+* You can add a timeout setting to the `approvalTask` specification. If the required number of approvals is not provided during this time period, the pipeline run fails.
+
+[id="cli-new-features-1-15_{context}"]
+=== CLI
+
+* With this update, the `tkn` command line utility supports the `-E` or `--exit-with-pipelinerun-error` option for the `pipeline showlog` command. With this option, the command line utility returns an error code of `0` if the pipeline run completed successfully, `1` if the pipeline run ended with an error, and `2` if the status of the pipeline run is unknown.
+
+* With this update, the `tkn` command line utility supports the `--label` option for the `bundle push` command. With this option, you can provide the value of a label in the `<label-name>=<value>` format; the utility adds the label to the OCI image that it creates. You can use this option several times to provide several labels for the same image.
+
+[id="pac-new-features-1-15_{context}"]
+=== {pac}
+
+* With this update, when using {pac}, you can set a `pipelinesascode.tekton.dev/on-comment` annotation on a pipeline run to start the pipeline run when a developer adds a matching comment to a pull request. This setting is supported only for pull requests and only for GitHub and GitLab repository providers.
```

Comment:
Based on https://pipelinesascode.com/docs/guide/authoringprs/#matching-a-pipelinerun-on-a-regexp-in-a-comment its a tech preview

```suggestion
* With this update, when using {pac}, you can set a `pipelinesascode.tekton.dev/on-comment` annotation on a pipeline run to start the pipeline run when a developer adds a matching comment to a pull request. This setting is supported only for pull requests and only for GitHub and GitLab repository providers. This feature is Technological preview only.
```
cc @chmouel